src/components/molecules/ScheduleEvent/ScheduleEvent.scss

```diff
@@ -0,0 +1,82 @@
+@import "scss/constants.scss";
+
+.ScheduleEvent {
+  display: flex;
+  position: absolute;
+  border-radius: 18px;
+  padding: 0 4px 0 12px;
+  cursor: pointer;
+  height: 60px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #3f3d42;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.65);
+  margin-top: 0.25rem;
+
+  &:hover {
+    background-color: #4c494f;
+    z-index: z(navbar__schedule-event--hover);
+  }
+
+  &--users {
+    color: #4c494f;
+    background-color: #ebebeb;
+
+    &:hover {
+      background-color: white;
+    }
+
+    .ScheduleEvent__bookmark {
+      color: black;
+    }
+  }
+
+  &--live {
+    background-color: #7c46fb;
+    color: white;
+
+    &:hover {
+      background-color: #8755fb;
+    }
+
+    .ScheduleEvent__bookmark {
+      color: white;
+    }
+  }
+
+  &__info {
+    font-size: 0.8rem;
+    overflow: hidden;
+  }
+
+  &__title {
+    white-space: pre-line;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
```

Comment:
looks like this is the way we should use `line-clamp` property according to 
https://css-tricks.com/almanac/properties/l/line-clamp/

the example from the article:
```css
.line-clamp {
  display: -webkit-box;
  -webkit-line-clamp: 3;
  -webkit-box-orient: vertical;  
  overflow: hidden;
}
```
and here is the support across the browsers: https://caniuse.com/css-line-clamp

although, it worth mentioning that this does not seem to work for Daniel. He uses Safari (haven't asked about the version), and he states that the width of event-boxes is calculated based on the width of the title rather than the duration of the event. 

This is my screenshot of the event which duration-width is shorter than the title-width.
![image](https://user-images.githubusercontent.com/63313732/118233223-9cc54980-b45f-11eb-9882-34f426476841.png)

It works for me in Firefox and Chrome, would be great if someone could check that on Safari as well. 
https://co-reality-staging--preview-pr-1302-dca4vx19.web.app/in/viktoryiaspartymap

Reply:
> looks like this is the way we should use line-clamp property according to https://css-tricks.com/almanac/properties/l/line-clamp/

Our babel/webpacker/etc setup as part of `create-react-app` should automatically add vendor prefixes as required based on our `browserlist` configuration; so I don't think we should have to manually add it here like this.

- https://create-react-app.dev/docs/post-processing-css/
  - > This project setup minifies your CSS **and adds vendor prefixes to it automatically through Autoprefixer so you don’t need to worry about it.**
    - https://github.com/postcss/autoprefixer
      - > Parse CSS and add vendor prefixes to rules by Can I Use

Reply:
hmmm looks like autoprefixer doesn't support that yet..
https://github.com/postcss/autoprefixer/issues/1322

created `@mixin line-clamp` here: https://github.com/sparkletown/sparkle/pull/1302/commits/f6812cfaa6a054cfddecaa3b9eadaa9459f9f5bd
can be useful I guess

Reply:
> hmmm looks like autoprefixer doesn't support that yet..
postcss/autoprefixer#1322

Can you please add an `@debt` comment where you defined `@mixin line-clamp` referencing that issue and saying to update/use that when available.

Reply:
added a comment: https://github.com/sparkletown/sparkle/pull/1302/commits/03d78303b9d16dfcdfde3ed631a87b04eec082ea